### DISCUSSION TITLE
Where is yocto_symrigid.h?

### DISCUSSION THREAD

**Discussion by redstonedash** [xelatihy/yocto-gl#1380] [Q&A]
A few websites link to this repository saying it contains a single file rigidbody simulator. However I can't find this file in the libs directory.

**Comment by xelatihy** [comment]
Was not supported any longer, so it has been removed.

**Reply by redstonedash** [reply]
Do you know what commit removed it?

**Reply by xelatihy** [reply]
Not really. But it was years ago.

**Reply by xvln** [reply]
quick search says https://github.com/xelatihy/yocto-gl/tree/aa1973d0f99c15528777413b4ab16eb91f50e15a is the last commit with it.

(it's still linked from https://github.com/nothings/single_file_libs fwiw)

**Reply by davidwarfogd** [reply]
Thank you very much @xvln
